signature/jws/envelope_test.go

```diff
@@ -563,7 +563,7 @@ func TestPayload(t *testing.T) {
 		checkNoError(t, err)
 
 		_, err = newEnv.Payload()
-		checkErrorEqual(t, "illegal base64 data at input byte 476", err.Error())
+		checkErrorEqual(t, "payload error: illegal base64 data at input byte 288", err.Error())
```

Comment:
I found this 288 strange... Can we do a fuzz check. For example, checking the error message contains the keyword

Reply:
Updated.
```
func checkErrorEqual(t *testing.T, want, got string) {
	if !strings.Contains(got, want) {
		t.Fatalf("want: %v got: %v\n", want, got)
	}
}
```